Source/Core/Updater/Main.cpp

```diff
@@ -670,6 +670,15 @@ void FatalError(const std::string& message)
 
 int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine, int nCmdShow)
 {
+  if (lstrlenW(pCmdLine) == 0)
+  {
+    MessageBox(nullptr,
+               L"This updater is not meant to be launched directly. Configure Auto-Update in "
+               "Dolphin's settings instead.",
```

Comment:
If you keep this line broken up, it should have the `L` prefix as well.

Reply:
No need to. It will still be a proper ``wchar_t*``.

Reply:
You're concatenating a `wchar_t*` with a `char*` there. Intentional or not, it should have the prefix (even if you'll never go and add characters that don't fit into a `char` later).

Reply:
Not according to Visual Studio and [cppreference](http://en.cppreference.com/w/cpp/language/string_literal):

>  If one of the strings has an encoding prefix and the other doesn't, the one that doesn't will be considered to have the same encoding prefix as the other. 

 Also: <img src="http://emojis.slackmojis.com/emojis/images/1515784750/3372/bikeshed.png?1515784750" height="24" width="24"></img>.

Reply:
But I want my shed painted green :^)
I'll just take cppreference's word for that; back in the day (which is already 15 years past) it had to be the same. VS docs don't really mention it at all, so I assumed it still is.

Reply:
The segment I was quoting has a ``(since C++11)`` note so that's not a surprise I guess.